docs/PreciseCaseReport.md

```diff
@@ -0,0 +1,2 @@
+# PRECISE Case Report
+See [wiki link](https://wiki.oicr.on.ca/display/GSI/PRECISE+Case+Report) for report documentation.
```

Comment:
Is there a reason not to include the documentation here like the other reports?

Reply:
PRECISE has its doc link to a wiki page, figured this would be of similar privacy level

Reply:
I'm not sure why that is. Maybe because project-specific information is revealed - types of samples received and labs they're received from. Whether it's on the wiki or in this repo, documentation should be in the same format as other reports though

Reply:
Rewrote the wiki page. Please let me know if there is any further information I should include on the page.